From 4e797a712f8244347e9396590725859a6e879add Mon Sep 17 00:00:00 2001
From: Tony Lin <tony.lin@freescale.com>
Date: Fri, 9 Sep 2011 11:45:11 +0800
Subject: [PATCH 0417/2463] ENGR00156314-1 [mx6q]gic: add comments to explain
 start irq offset value

to be more clear why we start irq offset from 29.
and list the irq ID table.

Signed-off-by: Tony Lin <tony.lin@freescale.com>
---
 arch/arm/mach-mx6/irq.c |    8 ++++++++
 1 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-mx6/irq.c b/arch/arm/mach-mx6/irq.c
index 4633607..cf7fdce 100644
--- a/arch/arm/mach-mx6/irq.c
+++ b/arch/arm/mach-mx6/irq.c
@@ -48,6 +48,14 @@ void mx6_init_irq(void)
 	struct irq_desc *desc;
 	unsigned int i;
 
+	/* start offset if private timer irq id, which is 29.
+	 * ID table:
+	 * Global timer, PPI -> ID27
+	 * A legacy nFIQ, PPI -> ID28
+	 * Private timer, PPI -> ID29
+	 * Watchdog timers, PPI -> ID30
+	 * A legacy nIRQ, PPI -> ID31
+	 */
 	gic_init(0, 29, IO_ADDRESS(IC_DISTRIBUTOR_BASE_ADDR),
 		IO_ADDRESS(IC_INTERFACES_BASE_ADDR));
 
-- 
1.7.7.4

